### PR TITLE
test changes for rubocop

### PR DESCRIPTION
**Pull request by grosser**
@sandlerr

### CODE REVIEW

File: test/controllers/sessions_controller_test.rb [hunk 1]
```diff
@@ -46,14 +46,14 @@
       )
     end
     let(:role_id) { Role::VIEWER.id }
-    let(:before_call) {  }
+    let(:before_call) {}
```
Comment (sandlerr) [inline test/controllers/sessions_controller_test.rb]: unused?

File: test/models/secret_storage_test.rb [hunk 2]
```diff
@@ -149,51 +184,76 @@
...
     describe ".write" do
-      it "writes a key with /secret" do
-        assert SecretStorage::HashicorpVault.write('production/foo/group/isbar/foo', {value: 'whatever'})
-        client.set.must_equal("secret/production/foo/group/isbar%2Ffoo" => {vault: 'whatever'})
+      before do
+        stub_request(:put, "https://127.0.0.1:8200/v1/secret/env/foo/bar/isbar%2Ffoo").
+          with(body: "{\"vault\":\"whatever\"}")
+      end
+
+      it "wirtes a key with /s" do
```
Comment (sandlerr) [inline test/models/secret_storage_test.rb]: writes

**Comment by sandlerr** [comment]
are changes in test_helper and secret_storage_test related to rubocop?

**Comment by grosser** [comment]
just merged a PR and then did a checkout from the rubocop branch ... so the changes got reverted ... better now :)

File: test/lib/samson/hooks_test.rb [hunk 3]
```diff
@@ -1,8 +1,6 @@
 require_relative '../../test_helper'
 
-unless defined?(Rake) # rake preloads all plugins
-  SingleCov.covered! uncovered: 13 unless defined?(Rake)
-end
+SingleCov.covered! uncovered: 13 unless defined?(Rake) || defined?(Rake) # rake preloads all plugins
```
Comment (sandlerr) [inline test/lib/samson/hooks_test.rb]: remove the last check?

File: test/models/git_repository_test.rb [hunk 4]
```diff
@@ -168,7 +168,7 @@
     it 'returns the tags repository' do
       create_repo_with_tags
       repository.clone!(mirror: true)
-      repository.tags.to_a.must_equal %w(v1 )
+      repository.tags.to_a.must_equal %w[v1 ]
```
Comment (sandlerr) [inline test/models/git_repository_test.rb]: odd whitespace?

File: test/lib/samson/hooks_test.rb (hunk 3 above)
Comment (grosser) [inline test/lib/samson/hooks_test.rb]: autocorrect ... :D

Comment (sandlerr) [inline test/lib/samson/hooks_test.rb]: ha, just realised we were already checking twice. Thanks rubocop :-)

File: test/models/git_repository_test.rb (hunk 4 above)
Comment (grosser) [inline test/models/git_repository_test.rb]: 🔥

**Comment by sandlerr** [comment]
+1

**Pull request merged**